uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseGridWidget.java

```diff
@@ -291,17 +303,45 @@ protected void drawWithoutTransforms(Context2D context,
 
         if (!isSelectionLayer) {
             //If there's no RenderingInformation the GridWidget is not visible
+            if (LOGGER.isLoggable(Level.FINEST)) {
```

Comment:
@manstis 
I understand reason of all those "Logger" (I had to do the same elsewhere) - but could you please just create a single method, invoked with the first part of the message to be logged (everything else is the same) ?

Reply:
Done.